plugins/org.python.pydev/src/org/python/pydev/editor/hover/PyEditorHoverConfigurationBlock.java

```diff
@@ -0,0 +1,1018 @@
+/**
+ * Copyright (c) 2016 by Brainwy Software LTDA. All Rights Reserved.
```

Comment:
If it was copied, you should keep the original copyright (the important thing is that it's EPL).